app/assets/javascripts/pretty-text/engines/discourse-markdown/emoji.js.es6

```diff
@@ -220,6 +220,14 @@ function applyEmoji(
 
       result.push(token);
 
+      if (i === 0 && i + offset === content.length) {
+        token.attrs.forEach(attr => {
+          if (attr[0] === "class") {
+            attr[1] = `${attr[1]} all-emoji`;
```

Comment:
I would have used `only-emoji`.

Reply:
I followed what mattermost was using on this, but sure only-emoji works for me too